setup.py

```diff
@@ -2,17 +2,9 @@
 import io
 from setuptools import find_packages, setup
 
-try:
-    # for pip >= 10
-    from pip._internal.req import parse_requirements
-except ImportError:
-    # for pip <= 9.0.3
-    from pip.req import parse_requirements
-
```

Comment:
nit: maybe only fix the bug in this PR?

Reply:
This prevented me to install it on my computer, it would be nice to include it in this PR